sdk/R/R/ArvadosR.R

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) The Arvados Authors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+#' @title ArvadosR
+#'
+#' @description
+#' \if{html}{\figure{dax.png}{options: style='float: right' alt='logo' width='120'}}
+#'
+#' Arvados is an open source platform for managing, processing, and sharing genomic and other large scientific and biomedical data. With Arvados, bioinformaticians run and scale compute-intensive workflows, developers create biomedical applications, and IT administrators manage large compute and storage resources.
+#'
+#' @author \itemize{
+#' \item Lucas Di Pentima
+#' \item Ward Vandewege
+#' \item Peter Amstutz
+#' \item fmuhic github}

```

Comment:
There is also a typo here that I hadn't noticed before. Line 16 should be removed and me and Piotr should be added,.